RealmSwift/SyncSubscription.swift

```diff
@@ -85,13 +89,19 @@ import Realm.Private
      - warning: This method may only be called during a write subscription block.
 
      - parameter type: The type of the object to be queried.
-     - parameter query: A query which will be used to modify the query.
+     - parameter query: A query which will be used to modify the existing query, if nil it will set the query to get all documents in the collection.
```

Comment:
Comma here should be a period followed by a new sentence.

A nearly always accurate rule is that if replacing a comma with a period gives two grammatically correct sentences, then the comma is incorrect.